libraries/helpers.rb

```diff
@@ -79,6 +90,7 @@ def default_major_version
       return '5.5' if trusty?
```

Comment:
This should be removed since you removed support for both 5.5 and 14.04. You should also remove the helper.